### PR TITLE
Added README.md

### PR DESCRIPTION
**Pull request by dgarcia360**
Before merging, please comment if you want to rename catapult-server to symbol-server (unsure).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,80 @@
...
 
-Detailed building instructions are [here](BUILDING.md).
+## Package Organization
+
+symbol-server code is organized as follows:
+
+| Folder name | Description |
+| -------------|--------------|
+|/extensions | Modules that add features to the bare symbol-server. They range from extensions that are critical, like consensus and networking to optional extensions like ZMQ messaging and other API conveniences. |
+| /external | Implementations of the hashing algorithms used.  |
```
Comment (BloodyRookie) [inline README.md]: not only hashing algorithm. probably should replace "hashing" with "cryptographic". @Jaguar0625 @gimer ?

Comment (gimer) [reply README.md]: reworded and pushed

**Comment by gimer** [comment]
merged to have it inside the build, we can reword/redo later

**Comment by gimer** [comment]
@dgarcia360 can you please re-open pull request, I messed up a bit

**Pull request merged**